### DISCUSSION TITLE
How can I resolve the "missing required error components, refreshing..." error in my application?

### DISCUSSION THREAD

**Discussion by avoesen** [neuefische/questions#815] [Web]
The server runs locally reliably and after a few minutes, this error message appears.

**Comment by NikRoe** [comment]
Hi Anna 👋 

I think this issue might be related to `node`.

I suggest you delete the `.next` folder and the `node_modules` in your project and afterwards run `npm i` and `npm run dev` again.

Let's see if that already does the trick and let me know otherwise ✌️

**Reply by avoesen** [reply]
I tried that yesterday and it worked for a bit longer, but the issue came back today. I'll report if and when it happens again ✌️

**Reply by NikRoe** [reply]
sounds like a plan 👌
